### PR TITLE
Google coding style Violates

### PR DESCRIPTION
**Pull request by subhamayghosh**

### CODE REVIEW

File: src/main/java/swaglabs/common/CommonMethods.java [hunk 1]
```diff
@@ -63,7 +65,7 @@ public <T extends WebElement> void click(T element) {
 	 * @param text
 	 * @param timeoutSeconds
 	 */
```
Comment (Priyanka2245968) [inline src/main/java/swaglabs/common/CommonMethods.java:67]: Please add more information here, and fix this typo.

**Review by Priyanka2245968 — CHANGES_REQUESTED** [review]
This is close to perfect! Please address the suggested inline change.

File: src/main/java/swaglabs/common/CommonMethods.java [hunk 2]
```diff
@@ -63,7 +65,7 @@ public <T extends WebElement> void click(T element) {
 	 * @param text
 	 * @param timeoutSeconds
 	 */
-	public <T extends WebElement> void waitForElementContainText(T element, String text, Integer timeoutSeconds) {
+	public static <T extends WebElement> void waitForElementContainText(T element, String text, Integer timeoutSeconds) {
```
Comment (Priyanka2245968) [inline src/main/java/swaglabs/common/CommonMethods.java:68]: Please add more information here, and fix this typo.

**Review by Priyanka2245968 — CHANGES_REQUESTED** [review]
This is close to perfect! Please address the suggested inline change.

File: src/main/java/swaglabs/common/CommonMethods.java [hunk 3]
```diff
@@ -101,5 +103,31 @@ public <T extends WebElement> void addToCartAndVerifyItemAdded(T addToCartElemen
...
+		if(buttonType.equalsIgnoreCase("Login")) waitForElementContainText(swagLabsHomePage.logoHeading, "Swag Labs", 10);
+		try {
+			loginPageCancelButton(1);
+		  } catch (Exception e) {}
+		
+		
+	}
+	//Test
+	//Test1
+	//Test2
```
Comment (Priyanka2245968) [inline src/main/java/swaglabs/common/CommonMethods.java:119]: Please add more information here, and fix this typo.

**Review by Priyanka2245968 — CHANGES_REQUESTED** [review]
This is close to perfect! Please address the suggested inline change.

File: src/main/java/swaglabs/common/CommonMethods.java [hunk 4]
```diff
@@ -101,5 +103,31 @@ public <T extends WebElement> void addToCartAndVerifyItemAdded(T addToCartElemen
...
+	public static void loginPageCancelButton(int input) {
+		switch(input) {
```
Comment (Priyanka2245968) [inline src/main/java/swaglabs/common/CommonMethods.java:121]: Please add more information here, and fix this typo.

**Review by Priyanka2245968 — CHANGES_REQUESTED** [review]
This is close to perfect! Please address the suggested inline change.

File: src/main/java/swaglabs/common/CommonMethods.java (hunk 4 above)
Comment (783967) [inline src/main/java/swaglabs/common/CommonMethods.java:121]: please correct the typo

**Review by 783967 — CHANGES_REQUESTED** [review]
This is close to perfect! Please address the suggested inline change.

File: src/test/java/swaglabs/tests/validateHomePageTest.java [hunk 5]
```diff
@@ -2,8 +2,14 @@
 
 import org.testng.annotations.Test;
 
+import swaglabs.common.CommonMethods;
+import static swaglabs.common.CommonMethods.loginPageButton;
 import swaglabs.components.DriverFactory;
```
Comment (783967) [inline src/test/java/swaglabs/tests/validateHomePageTest.java:7]: Please add more information here, and fix this typo.

File: src/main/java/swaglabs/common/CommonMethods.java [hunk 6]
```diff
@@ -19,6 +19,8 @@ public class CommonMethods extends BasePage {
 
 	private static WebDriver driver;
 	private static Logger logger;
+	private int         x;
+	int a, b,   c,  d;
```
Comment (783967) [inline src/main/java/swaglabs/common/CommonMethods.java:24]: Ensure the login button is visible.

**Review by 783967 — CHANGES_REQUESTED** [review]
Few review comments

**Review by 783967 — CHANGES_REQUESTED** [review]
Few review comments

**Review by 783967 — CHANGES_REQUESTED** [review]
Few review comments:

**Pull request closed without merging**